cs_simple_types.h

```diff
@@ -19,160 +19,162 @@ typedef enum {
 	// If you change this numbering, you must change the values in
 	// ValueTypes.td as well!
 	CS_DATA_TYPE_Other = 1, // This is a non-standard value
-	CS_DATA_TYPE_i1 = 2,	// This is a 1 bit integer value
```

Comment:
Are these lines formatted with the `.clang-format` of this PR?

Reply:
Yes

Reply:
Then please add the following options (if there is no objection from others):

For comment alignment
```
AlignTrailingComments:
  Kind: Always
  OverEmptyLines: 2
```

For escaped newlines (aligning them breaks for me. I guess because of the tabs).
```
AlignEscapedNewlines: DontAlign
```